csm/core/services/rgw/s3/users.py

```diff
@@ -43,3 +43,31 @@ async def create_user(self, **user_body):
         if isinstance(plugin_response, RgwError):
             self._handle_error(plugin_response)
         return plugin_response
+
+    @Log.trace_method(Log.DEBUG)
+    async def get_user(self, **request_body):
+        """
+        Method to get existing S3 IAM user.
+
+        :param **request_body: Request body kwargs
+        """
+        uid = request_body.get(const.RGW_JSON_UID)
+        Log.debug(f"Fetching S3 IAM user by uid = {uid}")
+        plugin_response = await self._s3_iam_plugin.execute(const.GET_USER_OPERATION, **request_body)
+        if isinstance(plugin_response, RgwError):
+            self._handle_error(plugin_response)
+        return plugin_response
+
+    @Log.trace_method(Log.DEBUG)
+    async def delete_user(self, **request_body):
+        """
+        Method to delete existing S3 IAM user.
+
+        :param **request_body: Request body kwargs
+        """
+        uid = request_body.get(const.RGW_JSON_UID)
+        Log.debug(f"Deleting S3 IAM user by uid = {uid}")
+        plugin_response = await self._s3_iam_plugin.execute(const.DELETE_USER_OPERATION, **request_body)
+        if isinstance(plugin_response, RgwError):
+            self._handle_error(plugin_response)
+        return plugin_response
```

Comment:
Medium: since the IAM CRUD methods don't differ much from each other, it might be a good idea to implement one generic function, that takes operation as a parameter.

Reply:
@avrnv Going forward, it is always better to have separate functions serving different purposes. 
Still I can make this change, please let me know.

Reply:
You might haven't got me right. You'll have separate functions, calling one.
Like  separate_function -> generic_function(specific parameters).

Reply:
done